### PR TITLE
Hide "Show all services" button on landing page if there are no more …

### PR DESCRIPTION
**Pull request by janimattiellonen**
…services available

Hide "Unused services" if there are no unused services available

### CODE REVIEW

File: assets/js/components/Tab/Service/Service.js [hunk 1]
```diff
@@ -73,12 +73,14 @@ class ServiceTab extends Component {
             </Col>
           </Row>
         </section>
-        <section>
-          <h2>
-            <FormattedMessage id="app.services.unconnectedHeading" />
-          </h2>
-          <ServiceList services={unusedServices} />
-        </section>
+        {unusedServices.length !== 0 && (
```
Comment (jaaniles) [inline assets/js/components/Tab/Service/Service.js]: could use isEmpty instead of array.length !== 0

Comment (janimattiellonen) [reply assets/js/components/Tab/Service/Service.js]: ah, there was another one

Comment (janimattiellonen) [reply assets/js/components/Tab/Service/Service.js]: Weird, I've fixed this and pushed

Comment (janimattiellonen) [reply assets/js/components/Tab/Service/Service.js]: Now it seems to be "fixed".

**Pull request merged**